compiler/src/iree/compiler/Codegen/TransformStrategies/GPU/AbstractGemmLikeStrategy.cpp

```diff
@@ -22,23 +23,22 @@ static llvm::cl::list<int64_t> clBlockTileSizes(
     "td-matmul-strategy-blk-sizes",
     llvm::cl::desc("block tile size for dims (x,y,z) for the transform "
                    "dialect matmul strategy"),
-    llvm::cl::list_init(ArrayRef<int64_t>{128, 128, 1}),
-    llvm::cl::CommaSeparated);
+    llvm::cl::list_init(ArrayRef<int64_t>{}), llvm::cl::CommaSeparated);
```

Comment:
I think we can drop the default empty/negative values and use `cl::opt::getNumOccurrences` to check if something has been set instead.

Where I'm going is since these values are meaningless, let us not set them to begin with.